CHANGELOG.md

```diff
@@ -55,7 +55,7 @@ _The old changelog can be found in the `release-2.6` branch_
   - Ask to confirm password on a newly generated PGP key
   - Prompt to push a key to the KeyStore when generated
   - Refuse to push an unsigned container unless overridden with `--allow-unauthenticated/-U` option
```

Comment:
Actually, the push flag is `--allow-unsigned`. And thats anther reason for https://github.com/sylabs/singularity/pull/3432.

Reply:
Is #3432 something we're trying to get into 3.2?

Reply:
Yeah, if we do anther rc we should include that PR.

Reply:
Doesn't seem like we'll do another rc for 3.2, @GodloveD can verify for sure. May be worth shimmying in though since this is really just a changed flag name

Reply:
We will do another rc if there is a show-stopping bug but have not seen one so far.